modules/core/src/v2/wallet.ts

```diff
@@ -484,6 +487,53 @@ export interface DownloadKeycardOptions {
   backupKeyID?: string;
 }
 
+// #region TSS interfaces
+
+export interface PrebuildTransactionWithIntentOptions {
+  reqId?: RequestTracer;
+  intentType?: string;
+  sequenceId?: string;
+  recipients?: {
+    address: string;
+    amount: string | number;
+  }[];
+  comment?: string;
+  memo?: Memo;
+  verification?: VerificationOptions;
+}
+
+export interface TSSSendManyOptions extends SendManyOptions {
+  reqId: RequestTracer;
+  intentType: string;
+}
+
+// find a better name
+export enum ShareKeyPosition {
+  USER = '1',
+  BACKUP = '2',
+  BITGO = '3',
+}
+
+// complete with more props
+export interface TransactionRequestJSON {
+  txRequestId: string;
+  unsignedTxs: string[];
+}
+
+export enum SignatureShareType {
```

Comment:
I was thinking here a kind of map structure with key value {[user,1], [backup,2], [bitgo, 3]} but the two enums is also valid. The only bad thing is your repeating names.

Reply:
This is the WP implementation of this interface, i didnt like it much neither
https://github.com/BitGo/bitgo-microservices/blob/e9e69bce28695c8ad545b61c6d93c5460ebd5ad6/packages/wallet-platform/app/models/v2/TransactionRequest/TransactionRequest.types.ts#L37-L39